lib/concentrate/vehicle_position.ex

```diff
@@ -98,17 +108,76 @@ defmodule Concentrate.VehiclePosition do
               second.previous_vehicle_schedule_adherence_string,
               first.previous_vehicle_schedule_adherence_string
             ),
-          route_id: first_value(second.route_id, first.route_id),
+          route_id:
+            swiftly_priority(
```

Comment:
Are there other associated fields that should be `swiftly_priority`? Headsign maybe?

Reply:
Added headsign. I think that's all, but we can always add more easily if we find the need.